src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1,16 +1,14 @@
 import { renderHook } from '@testing-library/react-hooks';
```

Comment:
It looks like this test file is still relying on the `CourseService` (at least from a mocking perspective).

Examples:
* `mockCourseService` is imported.

```jsx
jest.mock('../service', () => ({
  __esModule: true,
  default: jest.fn(() => mockCourseService),
}));
```